packages/openlegacy-cli/tools/chocolateyinstall.ps1

```diff
@@ -13,9 +13,12 @@ $packageArgs = @{
   checksum      = '5b2cd86f70446182e7efd2eb45ea2dfc88065ae42d25dfc47022c7d4fd6f3944'
   checksumType  = 'sha256' #default is md5, can also be sha1, sha256 or sha512
 }
-
 Install-ChocolateyZipPackage @packageArgs # https://chocolatey.org/docs/helpers-install-chocolatey-zip-package
-Install-ChocolateyPath "$toolsDir/ol/bin" 'Machine' # Machine will assert administrative rights
+Install-ChocolateyPath "$toolsDir\ol\bin" 'Machine' # Machine will assert administrative rights
```

Comment:
exact same changes have to be added to the TEMPLATE file  https://github.com/openlegacy/chocolatey-packages/blob/master/templates/openlegacy-cli/chocolateyinstall.ps1.erb otherwise your changes in this (current) file will get overwritten by the next CI run

Reply:
thanks, done.